### DISCUSSION TITLE
Cisco Webex

### DISCUSSION THREAD

**Discussion by dsuliman** [johnste/finicky#228] [General]
How to open the link directly for Cisco Webex without opening the browser like Zoom?

**Comment by johnste** [comment]
I can't help you there, not a user of Webex. Maybe someone else can.

**Comment by eddyg** [comment]
I've been wondering the same thing.

I ran this:

```shell
❯ mdls '/Applications/Cisco Webex Meetings.app' | grep kMDItemCF
kMDItemCFBundleIdentifier              = "com.cisco.webexmeetingsapp"
```

and tried adding this to my config:

```javascript
    {
      match: /(.+\.)?webex\.com\/(.+?)\/j\.php/,
      browser: "com.cisco.webexmeetingsapp"
    },
```

but I get a dialog (from Webex) saying "This feature is not supported in current version. Please upgrade to latest version."

I did find [this information](https://www.ketianyun.com/files/docfile/pdf/IOS_wbx_scheme.pdf) about iOS. Running `open wbx://cisco.webex.com/cisco` from the shell doesn't return an error (implying to me that `wbx://` is registered), but nothing happens either (the Webex app doesn't even come to the front). For what it's worth, `webexstart://` is also a valid URL handler, and what gets called when you're bounced through webex.com

For reference, the Webex API returns something like this for the meeting URL:

```xml
 <meet:meetingLink>https://sqdemo.dmz.webex.com/sqdemo/j.php?MTID=m0cdefe09626ba0bec956cdf9088bff55</meet:meetingLink>
```

**Comment by kiblik** [comment]
This works for me
```js
  rewrite: [{
    match: ({url}) => url.host.endsWith(".webex.com") && url.pathname.endsWith("/j.php"),
     url({url}) {
      return {
        protocol: "wbx",
        pathname: url.pathname.slice(0, url.pathname.length-6)
      }
    }
  }],
  handlers: [{
    match: ({ url }) => ( url.protocol == "wbx" ),
    browser: "com.webex.meetingmanager"
  }]
```

**Reply by ggiessen** [reply]
this works for me

```
    {
      match: ({ url }) => url.host.includes("webex.com" ),
      url: ({url}) =>
        ({...url, protocol: "wbx"}),
      browser: "com.webex.meetingmanager"
    }
```

**Reply by SoCuul** [reply]
@sindresorhus

**Comment by ajostergaard** [comment]
I get this in Firefox, does this help?

![image](https://github.com/user-attachments/assets/fbfb9c6b-586b-4cbc-9295-995e9399baed)
